Examples/JamitFoundationExample/App/Sources/ScrollView/MyScrollViewController.swift

```diff
@@ -0,0 +1,20 @@
+//  Copyright © 2020 Jamit Labs GmbH. All rights reserved.
+
+import JamitFoundation
+import UIKit
+
+final class MyScrollViewController: StatefulViewController<MyScrollViewControllerViewModel>{
+    @IBOutlet private var textLabel: UILabel!
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        title = "MyScrollViewController"
```

Comment:
Title should be passed through the viewModel

Reply:
Are you sure about that? This is only used to show through the view controller title which view controller this is so you directly see how the things work. In my opinion this would only blow up the view model with information not really necessary. 

As it is an example I would suggest leaving it as is.

Reply:
Thats ok for me!